Tools/mavlink_shell.py

```diff
@@ -15,8 +15,8 @@
 
 try:
     from pymavlink import mavutil
-except ImportError as e:
-    print("Failed to import pymavlink: " + e)
+except ImportError:
+    print("Failed to import pymavlink")
```

Comment:
You can just cast it explicitly:
```suggestion
    print("Failed to import pymavlink": " + str(e))
```

Reply:
I was wondering if that would be enough. I tried it and yes, it seems to works. I'll change it, thanks!

Reply:
Fixed and force pushed.